### DISCUSSION TITLE
Docker image crashes, when run search/query

### DISCUSSION THREAD

**Discussion by Jaroslove** [milvus-io/milvus#27068] [Q&A and General discussion] [answered]
Hi Team,

I use Docker image quay.io/coreos/etcd:v3.5.5, minio/minio:RELEASE.2023-03-20T20-16-18Z, milvusdb/milvus:v2.3.0
call any search/query methods via dotnet client Milvus.Client --version 2.2.2-preview.5
and I get error [error="find no available rootcoord, check rootcoord state"]
after this my docker image crashes.
I use the save code as pointed here https://github.com/milvus-io/milvus-sdk-csharp/blob/main/docs/readme.md
Could you tell me about this error, please?

**Comment by yhmo** [comment]
"find no available rootcoord, check rootcoord state" ------------------- typically, this is caused by unhealthy etcd service, the root coordinator of milvus could not work without etcd.

Try the command  `docker ps` to list the status of the containers. Is the etcd container alive?

**Reply by Jaroslove** [reply]
Yes, etcd and minio are alive, but standalone is not. I downgraded the version of image from milvusdb/milvus:v2.3.0 to milvusdb/milvus:v2.2.14 and It works now.

**Reply by yhmo** [reply]
Your milvus previous version is v2.2.x, you upgraded it to v2.3.0 and it failed to start. Then you downgraded it to v2.2.14?

**Reply by Jaroslove** [reply]
My first version was v2.3.0 it started in docker but when i did any search/query was failed and docker stopped the container, then I downgraded it to v2.2.14 and it works fine for now.

**Reply by yhmo** [reply, marked as answer]
Our QA team has tested a lot of case that search/query after upgrading from v2.2.x to v2.3.0. They didn't encounter such error. 
What error is returned by search/query after upgrading? Any log for that?

**Reply by Jaroslove** [reply]
I upgraded my docker and v2.3.0 is running well now. I guess the error was because I used the old version of docker.
